examples/jsm/loaders/USDZLoader.js

```diff
@@ -368,7 +368,7 @@ class USDZLoader extends Loader {
 			if ( 'int[] faceVertexIndices' in data ) {
 
 				const indices = JSON.parse( data[ 'int[] faceVertexIndices' ] );
-				geometry.setIndex( new BufferAttribute( new Uint16Array( indices ), 1 ) );
+				geometry.setIndex( new BufferAttribute( new Uint32Array( indices ), 1 ) );
```

Comment:
I think you can just do this:
```js
geometry.setIndex( indices ); 
```
You can let `BufferGeometry.setIndex()` pick the right data type.

Reply:
It is now changed and I did test it as working.

This will be the only code change I would propose for this PR.

There is a way to also update textures which I might suggest to @hybridherbst. These changes are currently available in my viewers but my files are somewhat customized.